rspace/src/main/scala/coop/rchain/rspace/history/ITrieStore.scala

```diff
@@ -0,0 +1,18 @@
+package coop.rchain.rspace.history
+
+import scala.concurrent.SyncVar
+
+trait ITrieStore[T, K, V] {
+
+  private[rspace] def createTxnRead(): T
+
+  private[rspace] def createTxnWrite(): T
+
+  private[rspace] def withTxn[R](txn: T)(f: T => R): R
+
+  private[rspace] val workingRootHash: SyncVar[Blake2b256Hash] = new SyncVar[Blake2b256Hash]()
```

Comment:
This doesn't seem to be used anywhere

Reply:
Yes, it's part of the work that is in flight and should be landing in a PR later today.